.github/workflows/python-push.yml

```diff
@@ -104,68 +76,9 @@ jobs:
       run: |
         make test-cov
 
-    - name: Upload artifact
-      if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8' }}
-      uses: actions/upload-artifact@v2
-      with:
-        name: coverage
-        path: coverage.xml
-
-  sonar:
-    if: github.repository == 'IBM/compliance-trestle-fedramp'
-    runs-on: ubuntu-latest
-    needs: test
-    steps:
-    - name: Don't mess with line endings
-      run: |
-        git config --global core.autocrlf false
-    - name: Don't mess with line endings
-      run: |
-        git config --global core.autocrlf false
-    - uses: actions/checkout@v2
-      with:
-        submodules: true
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - uses: actions/cache@v2
-      with:
-        path: ~/.cache/pip
-        key: ubuntu-latest-3.9-pip-${{ hashFiles('setup.cfg') }}
-        restore-keys: |
-          ubuntu-latest-3.9-pip-
-    - name: Install build tools
-      run: |
-        make develop
-    - name: Get coverage
-      uses: actions/download-artifact@v2
-      with:
-        name: coverage
-    - name: SonarCloud Scan
-      uses: SonarSource/sonarcloud-github-action@master
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      with:
-        args: >
-          -Dsonar.python.coverage.reportPaths=coverage.xml
-          -Dsonar.tests=tests/
-          -Dsonar.sources=trestle_fedramp/ 
-          -Dsonar.python.version=3.8
-          -Dsonar.projectKey=compliance-trestle-fedramp
-          -Dsonar.organization=compliance-trestle
-    - name: SonarQube Quality Gate check
-      uses: sonarsource/sonarqube-quality-gate-action@master
-      # Force to fail step after specific time
-      timeout-minutes: 5
-      env:
-       SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-
```

Comment:
@jpower432 Why have you removed Sonar. I thought this was working now

Reply:
@vikas-agarwal76 You are correct. The sonar job is working, but I updated the workflow here to match the one in `compliance-trestle` where `develop` is the main branch and additions to that branch are what is gated by the sonar job. Though multiple long lived branches are support so main could be scanned as well.